terraform/Registrator.json.template

```diff
@@ -1,6 +1,7 @@
 {
   "name": "registrator",
   "image": "gliderlabs/registrator:latest",
+  "privileged": true,
```

Comment:
why does registrator also need to be `privileged`?

Reply:
because it already requires mounting the docker socket http://gliderlabs.github.io/registrator/latest/ (see their runtime params), which is effectively privileged behavior anyway. I prefer making this overall behavior more explicit.